aten/src/ATen/native/sparse/cuda/SparseCUDATensorMath.cu

```diff
@@ -495,7 +495,7 @@ SparseTensor& mul_out_sparse_cuda(const SparseTensor& t_, const SparseTensor& sr
 
   Tensor r_values_ = new_values_with_size_of(t_values_, max_nnz).zero_();
 
-  int64_t valueSize = t_values_.stride(0);
+  int64_t valueSize = t_values_.size(0);
```

Comment:
So, basically, we fail with 0-dim tensors. I guess now we also get a speed-up for free, because before the block size was of size just 1 with non-scalar index Tensor :)

Reply:
`valueSize` is solely used for deciding the grid/block size, so it should be still correct provided that indices are handled correctly.

Reply:
It was a choice between 1 and 4 only.

Reply:
Regarding tests, I fail to see an explicit test for `mul` in sparse. Does OpInfo work with that?
Edit: found `test_basic_ops`...

Reply:
You are right, sorry for the confusion. I retract my words, then we have a choice between 4 and `values.numel() / values.size(0)`. Since `values` is contiguous, we can still "preserve" the previous behavior of `stride(0)` but use a combination of `numel` and `size(0)` instead. What do you think?

Reply:
I decided to keep the previous behavior, but made sure that `valueSize` is at least `min(1, stride(0))`. This is, I guess, will be safest option.